hydroshare/urls.py

```diff
@@ -23,7 +24,6 @@
     # Change the admin prefix here to use an alternate URL for the
     # admin interface, which would be marginally more secure.
     url("^admin/", include(admin.site.urls)),
-    url(r'^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
@xuezhaokun You did not address the comment above. Please merge or rebase your branch with latest develop and bring your branch up to date to avoid the problem of removing others' work which already got into develop but not merged into your branch. This is an important prerequisite before this branch can be merged, so please take care of it.